src/components/NavBar/index.tsx

```diff
@@ -59,7 +59,13 @@ const NavBar: React.FC = () => {
             by <span>iterative.ai</span>
```

Comment:
Can we add a `justify-content: center` to the `styles.nav` class? 

Without `justify-content: center`:

<img width="378" alt="image" src="https://user-images.githubusercontent.com/43496356/176672128-48024839-e853-48b4-85a1-6598fe93b593.png">

With `justify-content: center`: 

<img width="466" alt="image" src="https://user-images.githubusercontent.com/43496356/176672203-705695bf-2906-4ff9-8674-0f6ce4e6fcb4.png">